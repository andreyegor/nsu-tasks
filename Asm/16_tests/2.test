beetle
beet
better
beauty
bell