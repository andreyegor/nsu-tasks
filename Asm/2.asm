.include "things.asm"

main:
	li t2 '\n'
	read
	beq a0 t2 quit
	mv t0 a0
	writi '\n'
	mv a0 t0
	write
	addi t0 a0 1
	writi '\n'
	mv a0 t0
	write
	writi'\n'
	j main
	

quit:
	exit
