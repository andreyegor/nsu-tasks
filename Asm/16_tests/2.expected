beauty
beet
beetle
bell
better